src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddFileMetadataToDepositSpec.scala

```diff
@@ -203,28 +264,48 @@ class AddFileMetadataToDepositSpec extends UnitSpec with BeforeAndAfter {
 
   it should "produce the xml for all the files" in {
     val dataset = mutable.HashMap(
-      "DATASET" -> List(datasetID, datasetID),
-      "AV_FILE" -> List("ruimtereis01/reisverslag/centaur.mpg", "")
+      "DATASET" -> List(datasetID, datasetID, datasetID),
+      "SF_DOMAIN" -> List("dans", "", ""),
+      "SF_USER" -> List("janvanmansum", "", ""),
+      "SF_COLLECTION" -> List("Jans-test-files", "", ""),
+      "SF_ACCESSIBILITY" -> List("RESTRICTED_GROUP", "", ""),
+      "AV_FILE" -> List("ruimtereis01/reisverslag/centaur.mpg", "ruimtereis01/reisverslag/centaur.mpg", "ruimtereis01/path/to/a/random/sound/chicken.mp3"),
+      "AV_FILE_TITLE" -> List("video about the centaur meteorite", "", "our daily wake up call"),
+      "AV_SUBTITLES" -> List("ruimtereis01/reisverslag/centaur.srt", "ruimtereis01/reisverslag/centaur-nederlands.srt", ""),
+      "AV_SUBTITLES_LANGUAGE" -> List("en", "nl", "")
     )
     AddFileMetadataToDeposit(1, (datasetID, dataset)).execute() shouldBe a[Success[_]]
 
-    Utility.trim(XML.loadFile(stagingFileMetadataFile(datasetID)))
-      .child
-      .map(node => (node \@ "filepath", node.child.filter(_.label == "format").head.text)) should {
-      have length 10 and
-        contain allOf(
-        ("data/ruimtereis01_verklaring.txt", "text/plain"),
-        ("data/reisverslag/deel01.docx", "application/vnd.openxmlformats-officedocument.wordprocessingml.document"),
-        ("data/reisverslag/deel01.txt", "text/plain"),
-        ("data/reisverslag/deel02.txt", "text/plain"),
-        ("data/reisverslag/deel03.txt", "text/plain"),
-        ("data/path/to/a/random/video/hubble.mpg", "video/mpeg"),
-        ("data/path/to/a/random/sound/chicken.mp3", "audio/mpeg"),
-        ("data/reisverslag/centaur.mpg", "video/mpeg"),
-        ("data/reisverslag/centaur.srt", "text/plain"),
-        ("data/reisverslag/centaur-nederlands.srt", "text/plain")
-      )
-    }
+    val expected = XML.loadFile(new File(getClass.getResource("/allfields/output/input-ruimtereis01/bag/metadata/files.xml").toURI))
```

Comment:
That is still no explicit confirmation. It might be the title is only meant to store in the file-metadata and I should stick with the numbers. Currently I implemented the following transition (no PR yet, I'm wainting for https://github.com/DANS-KNAW/easy-ingest-flow/pull/13 to be merged to not overwelm the reviewers even more)
```scala
    writeStringToFile(
      deposit.resolve("bag/metadata/files.xml").toFile,
      """<?xml version="1.0" encoding="UTF-8"?>
        |<files xmlns:dcterms="http://purl.org/dc/terms/">
        |  <file filepath="data/path/to/a/random/sound/chicken.mp3">
        |    <dcterms:format>audio/mpeg</dcterms:format>
        |  </file>
        |  <file filepath="data/path/to/a/random/video/hubble.mpg">
        |    <dcterms:format>video/mpeg</dcterms:format>
        |  </file>
        |  <file filepath="data/reisverslag/centaur-nederlands.srt">
        |    <dcterms:format>text/plain</dcterms:format>
        |  </file>
        |  <file filepath="data/reisverslag/centaur.mpg">
        |    <dcterms:format>video/mpeg</dcterms:format>
        |  </file>
        |  <file filepath="data/reisverslag/centaur.srt">
        |    <dcterms:format>text/plain</dcterms:format>
        |  </file>
        |</files>""".stripMargin
    )

    createDisseminationCopies shouldBe a[Success[_]]

    readFileToString(springfieldInbox.resolve("springfield-actions.xml").toFile, XML_FILE_ENCODING) shouldBe
      """<?xml version="1.0" encoding="UTF-8"?>
        |<add target="/domain/dans/user/somebody/collection/test1/presentation/testA">
        |  <video src="0" target="data/path/to/a/random/sound/chicken.mp3"/>
        |  <video src="1" target="data/path/to/a/random/video/hubble.mpg"/>
        |  <video src="2" target="data/reisverslag/centaur.mpg"/>
        |  <video src="data/reisverslag/centaur.mpg" target="2" subtitles="data/reisverslag/centaur-nederlands.srt"/>
        |  <video src="data/reisverslag/centaur.mpg" target="2" subtitles="data/reisverslag/centaur.srt"/>
        |</add>""".stripMargin
```